source/opt/basic_block.h

```diff
@@ -38,12 +38,14 @@ class BasicBlock {
   using const_iterator = UptrVectorIterator<Instruction, true>;
 
   // Creates a basic block with the given starting |label|.
-  inline explicit BasicBlock(std::unique_ptr<Instruction> label);
+  inline explicit BasicBlock(std::unique_ptr<Instruction> _label);
```

Comment:
The scoping rules handle it correctly, but produces warning:
/ssd/go/src/github.com/google/gapid/third_party/SPIRV-Tools/source/opt/basic_block.h:71:65: error: declaration of ‘label’ shadows a member of 'this' [-Werror=shadow]

I do not like the underscore either though.  How about upper-casing the accessor method instead?